lib/public/views/Logs/Logs.js

```diff
@@ -50,14 +55,89 @@ export default class Overview extends Observable {
         this.notify();
     }
 
+    /**
+     * Create the log with the variables set in the model, handle errors appriopately
+     * @returns {undefined}
+     */
+    async createLog() {
+        const response = await fetchClient('api/logs', {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                title: this.getTitle(),
+                text: this.getText(),
+            }),
+        });
+
+        const result = await response.json();
+
+        if (result.data) {
+            const logId = await result.data.id;
+            this.fetchedFromCreation = true;
+            await this.model.router.go(`/?page=entry&id=${logId}`);
+            this.error = false;
+        } else {
+            this.error = true;
+            this.errorMessage = result.errors[0].detail;
+            this.notify();
+        }
+    }
+
+    /**
+     * Returns the set errormessage
+     * @returns {String} Returns the set errormessage
+     */
+    getErrorMessage() {
+        return this.errorMessage;
+    }
+
+    /**
+     * Set the title paramter in the model
+     * @param {String} title Received string from the view
+     * @returns {undefined}
+     */
+    setTitle(title) {
+        this.title = title;
+        this.notify();
+    }
+
+    /**
+     * Set the text paramter in the model
+     * @param {String} text Received string from the view
+     * @returns {undefined}
+     */
+    setText(text) {
+        this.text = text;
+        this.notify();
+    }
+
+    /**
+     * Get the title value
+     * @returns {String} the title in the model
+     */
+    getTitle() {
+        return this.title;
+    }
+
+    /**
+     * Get the text value
+     * @returns {String} the text in the model
+     */
+    getText() {
+        return this.text;
+    }
+
     /**
      * Retrieve a specified log from the API
      * @param {Number} id The ID of the log to be found
      * @returns {undefined} Injects the data object with the response data
      */
     async fetchOneLog(id) {
-        // Do not call this endpoint again if we already have all the logs
```

Comment:
Comment indent should be aligned with the code.

Reply:
Locally the comment shows the right indentation, don't know why it shows up like this on github

Reply:
![image](https://user-images.githubusercontent.com/14125058/82652730-65420980-9c1e-11ea-9a3a-a41eecaa4069.png)